cmd/root.go

```diff
@@ -38,6 +40,7 @@ var RootCmd = &cobra.Command{
 func init() {
 	RootCmd.PersistentFlags().BoolVarP(&keep, "keep", "k", false, "Do not remove container image")
 	RootCmd.PersistentFlags().BoolVarP(&reboot, "reboot", "r", false, "reboot if changed")
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
```

Comment:
This single line took me a ridiculous amount of time to figure out... The whole flag parsing/command handling ecosystem in golang is pretty insane.